### ISSUE TITLE
Script tag dialog is too wordy

### ISSUE THREAD

**Issue by iwehrman** [adobe/brackets-edge-web-fonts#114]
As discussed in #112 with @larz0 and @njx, the current script tag dialog is rather wordy: 

![3922](http://f.cl.ly/items/1W0C01000K370k01342H/Screen%20Shot%202013-05-20%20at%2010.49.32%20AM.png)

@larz0 suggested rewording thusly: 

```
 Add this JavaScript tag to the <head> of your website's HTML that
 reference this CSS file:
```

@iwehrman felt the need to add his own suggestion (which he now feels isn't quite right, but isn't yet ready to offer up something better): 

```
Edge Web Fonts are loaded with JavaScript. To use the fonts you've
chosen, paste the following script tag into HTML files that reference
this CSS file.
```

Perhaps PM @ryanstewart has some thoughts here.

**Comment by ryanstewart** [comment]
What about: 

`To ensure the fonts on your site load correctly add this script tag to the <head> of any HTML files that reference this CSS file.`

 :question: 

Should there be a comma in there somewhere :question:

**Comment by larz0** [comment]
Comma between "correctly" and "add"?

**Comment by larz0** [comment]
BTW the close icon on the top right needs to center vertically in the modal header. Just realized it's been like that since the beginning.

**Comment by ryanstewart** [comment]
So this?

To ensure the fonts on your site load correctly, add this script tag to the `<head>` of any HTML files that reference this CSS file.

Does that work for people?

**Comment by iwehrman** [comment]
@larz0 I'll look into vertically aligning the close button.
@ryanstewart That wording looks good to me.

**Comment by iwehrman** [comment]
@larz0 It looks like Brackets is still using the Bootstrap-default styling of `<code>` blocks with a light-orange background: 

![head](http://f.cl.ly/items/1B0i2P051e2r3f2J0C2k/Screen%20Shot%202013-05-21%20at%2012.38.10%20PM.png)

Think we should submit a Brackets pull to fix that? Otherwise I can just style it manually with a monospace font.

**Comment by larz0** [comment]
@iwehrman I can't think of a reason why we would want to highlight an inline code snippet in Brackets right now because the font style is already the differentiator so let's submit a Brackets pull to remove the background color. (Hope I don't get busted this time lol).

**Comment by iwehrman** [comment]
OK, cool. Who wants it? Flip for it?

**Comment by larz0** [comment]
(I already have too many pulls that won't make it into sprint.)

**Comment by iwehrman** [comment]
I'll lob another one over the fence.

**Comment by iwehrman** [comment]
Tracking https://github.com/adobe/brackets/pull/3949

**Comment by couzteau** [comment]
merged fix- closing
